integration/test/basic/main_test.go

```diff
@@ -36,12 +40,22 @@ var (
 )
 
 func init() {
+	ctx := context.Background()
 	var err error
 
+	var latestRelease string
 	{
-		tarballURL = os.Getenv(envVarTarballURL)
-		if tarballURL == "" {
-			panic(fmt.Sprintf("env var '%s' must not be empty", envVarTarballURL))
+		latestRelease, err = appcatalog.GetLatestVersion(ctx, catalogURL, appName)
+		if err != nil {
+			panic(err.Error())
+		}
+	}
+
+	{
+		version := fmt.Sprintf("%s-%s", latestRelease, env.CircleSHA())
+		tarballURL, err = appcatalog.NewTarballURL(testCatalogURL, appName, version)
+		if err != nil {
+			panic(err.Error())
 		}
```

Comment:
We use the appcatalog library to create the tarballURL. Before this was an env var.